README.md

```diff
@@ -50,7 +53,10 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+castle[:guests] << residents[1]
+residents.delete("Belle")
+
```

Comment:
castle and residents are undefined as variables. They only have values as symbols in the `town` hash
```ruby
town[:castle].push(town[:residents].delete("belle"))
```